modules/images-update-global-pull-secret.adoc

```diff
@@ -25,18 +25,43 @@ As of {product-title} 4.7, changes to the global pull secret no longer trigger a
 
 .Prerequisites
 
-* You have a new or modified pull secret file to upload.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
+
 .Procedure
+To update the global pull secret, complete the following steps:
```

Comment:
I suggest removing this line - it doesn't add new information, and the [supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences) says:
> Do not use a lead-in sentence in the Prerequisites or Procedure sections of a module unless it is necessary to aid navigation or add clarity.